tests/manual-test-cases/Group5-Functional-Tests/5-6-2-VSAN-Complex.robot

```diff
@@ -25,6 +25,12 @@ VSAN Complex Setup
     ${name}=  Evaluate  'vic-vsan-complex-' + str(random.randint(1000,9999))  modules=random
     Set Suite Variable  ${user}  %{NIMBUS_USER}
     ${out}=  Deploy Nimbus Testbed  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}  --plugin testng --vcfvtBuildPath /dbc/pa-dbc1111/mhagen/ --noSupportBundles --vcvaBuild ${VC_VERSION} --esxPxeDir ${ESX_VERSION} --esxBuild ${ESX_VERSION} --testbedName vic-vsan-complex-pxeBoot-vcva --runName ${name}
+
+    # Finish testbed deploy
+    ${output}=  Wait For Process  ${out}  timeout=70 minutes  on_timeout=terminate
```

Comment:
${out} is not a PID AFAIK...

Reply:
hmmm okay, is this kind of Wait not appropriate here then?

Reply:
@mhagen-vmware I should take out this Wait For Process then, yes?